appveyor.yml

```diff
@@ -7,24 +7,37 @@ environment:
   VCINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
   VS150COMNTOOLS: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
   VSVARS32FILE: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars32.bat
-  MSVC_VER: 14.12.25827
+  # MSVC_VER: 14.12.25827
   APPVEYOR_CACHE_ENTRY_ZIP_ARGS: "-t7z -m0=lzma -mx=9"
+  matrix:
+  - ARCHITECTURE: x64
+    MSVC_VER: 15
```

Comment:
I believe we discovered on the mailing list that OpenJFX gradle build files do not expect the actual MSVC version number, but some weird pseudo-version which happened to be `14.12.25827` at the time of my making the appveyor PR. I did not find a way to fetch this version from the Appveyor environment, so it would need to be changed if Appveyor updates their environment. I believe having MSVC_VER set to 15 will never work with building OpenJFX on Windows.